test/release/index.js

```diff
@@ -304,63 +264,17 @@ document.addEventListener('DOMContentLoaded', function() {
             const js = version === 'latest' ? jsLatest.href : 'https://api.mapbox.com/mapbox-gl-js/' + version + '/mapbox-gl.js';
             const css = version === 'latest' ? cssLatest.href : 'https://api.mapbox.com/mapbox-gl-js/' + version + '/mapbox-gl.css';
 
+            const versionLibRegex = /https:\/\/api\.mapbox\.com\/mapbox-gl-js\/v[0-9]\.[0-9]\.[0-9]\/mapbox-gl\.js/g;
+            const versionCSSRegex = /https:\/\/api\.mapbox\.com\/mapbox-gl-js\/v[0-9]\.[0-9]\.[0-9]\/mapbox-gl\.css/g;
```

Comment:
Let's add `+` so this is compatible for, e.g. `2.10.1`, just in case. See: [test regex](https://scriptular.com/#https%3A%5C%2F%5C%2Fapi%5C.mapbox%5C.com%5C%2Fmapbox-gl-js%5C%2Fv%5B0-9%5D%2B%5C.%5B0-9%5D%2B%5C.%5B0-9%5D%2B%5C%2Fmapbox-gl%5C.css%7C%7C%7C%7Cg%7C%7C%7C%7C%5B%22%22%2C%22%3C!DOCTYPE%20html%3E%20%3Chtml%3E%20%3Chead%3E%20%3Cmeta%20charset%3D%5C%22utf-8%5C%22%3E%20%3Ctitle%3EMapbox%20GL%20JS%20example%3C%2Ftitle%3E%20%3Cmeta%20name%3D%5C%22viewport%5C%22%20content%3D%5C%22initial-scale%3D1%2Cmaximum-scale%3D1%2Cuser-scalable%3Dno%5C%22%3E%20%3Cmeta%20name%3D%5C%22robots%5C%22%20content%3D%5C%22noindex%2C%20nofollow%5C%22%3E%20%3Clink%20href%3D%5C%22https%3A%2F%2Fapi.mapbox.com%2Fmapbox-gl-js%2Fv2.2.10%2Fmapbox-gl.css%5C%22%20rel%3D%5C%22stylesheet%5C%22%3E%20%3Cscript%20src%3D%5C%22https%3A%2F%2Fjs.sentry-cdn.com%2Fb4e18cb1943f46289f67ca6a771bd341.min.js%5C%22%20crossorigin%3D%5C%22anonymous%5C%22%3E%3C%2Fscript%3E%20%3Cscript%20src%3D%5C%22https%3A%2F%2Fapi.mapbox.com%2Fmapbox-gl-js%2Fplugins%2Fmapbox-gl-instrumentile%2Fv3.0.0%2Fmapbox-gl-instrumentile.js%5C%22%20crossorigin%3D%5C%22anonymous%5C%22%3E%3C%2Fscript%3E%20%3Cscript%20src%3D%5C%22https%3A%2F%2Fapi.mapbox.com%2Fmapbox-gl-js%2Fv2.2.0%2Fmapbox-gl.js%5C%22%3E%3C%2Fscript%3E%20%3Cstyle%3Ebody%7Bmargin%3A0%3Bpadding%3A0%7D%23map%7Bposition%3Aabsolute%3Btop%3A0%3Bbottom%3A0%3Bwidth%3A100%25%7D%3C%2Fstyle%3E%20%3Cscript%3Emapboxgl.accessToken%3D%5C%22pk.eyJ1IjoiZXhhbXBsZXMiLCJhIjoiY2p0MG01MXRqMW45cjQzb2R6b2ptc3J4MSJ9.zA2W0IkI0c6KaAhJfk9bWg%5C%22%3C%2Fscript%3E%20%3C%2Fhead%3E%20%3Cbody%3E%20%3Cdiv%20id%3D%5C%22map%5C%22%3E%3C%2Fdiv%3E%20%3Cscript%3Evar%20map%3Dnew%20mapboxgl.Map(%7Bcontainer%3A%5C%22map%5C%22%2Cstyle%3A%5C%22mapbox%3A%2F%2Fstyles%2Fmapbox%2Flight-v10%5C%22%2Ccenter%3A%5B-96%2C37.8%5D%2Czoom%3A2%7D)%3Bmap.on(%5C%22load%5C%22%2C(function()%7Bmap.loadImage(%5C%22https%3A%2F%2Fdocs.mapbox.com%2Fmapbox-gl-js%2Fassets%2Fcustom_marker.png%5C%22%2C(function(t%2Ce)%7Bif(t)throw%20t%3Bmap.addImage(%5C%22custom-marker%5C%22%2Ce)%2Cmap.addSource(%5C%22points%5C%22%2C%7Btype%3A%5C%22geojson%5C%22%2Cdata%3A%7Btype%3A%5C%22FeatureCollection%5C%22%2Cfeatures%3A%5B%7Btype%3A%5C%22Feature%5C%22%2Cgeometry%3A%7Btype%3A%5C%22Point%5C%22%2Ccoordinates%3A%5B-77.03238901390978%2C38.913188059745586%5D%7D%2Cproperties%3A%7Btitle%3A%5C%22Mapbox%20DC%5C%22%7D%7D%2C%7Btype%3A%5C%22Feature%5C%22%2Cgeometry%3A%7Btype%3A%5C%22Point%5C%22%2Ccoordinates%3A%5B-122.414%2C37.776%5D%7D%2Cproperties%3A%7Btitle%3A%5C%22Mapbox%20SF%5C%22%7D%7D%5D%7D%7D)%2Cmap.addLayer(%7Bid%3A%5C%22points%5C%22%2Ctype%3A%5C%22symbol%5C%22%2Csource%3A%5C%22points%5C%22%2Clayout%3A%7B%5C%22icon-image%5C%22%3A%5C%22custom-marker%5C%22%2C%5C%22text-field%5C%22%3A%5B%5C%22get%5C%22%2C%5C%22title%5C%22%5D%2C%5C%22text-font%5C%22%3A%5B%5C%22Open%20Sans%20Semibold%5C%22%2C%5C%22Arial%20Unicode%20MS%20Bold%5C%22%5D%2C%5C%22text-offset%5C%22%3A%5B0%2C1.25%5D%2C%5C%22text-anchor%5C%22%3A%5C%22top%5C%22%7D%7D)%7D))%7D))%3C%2Fscript%3E%20%3C%2Fbody%3E%20%3Cscript%3Eif(window.map%20instanceof%20mapboxgl.Map)var%20i%3Dnew%20instrumentile(map%2C%7Btoken%3A%5C%22pk.eyJ1IjoiZXhhbXBsZXMiLCJhIjoiY2p0MG01MXRqMW45cjQzb2R6b2ptc3J4MSJ9.zA2W0IkI0c6KaAhJfk9bWg%5C%22%2Capi%3A%5C%22https%3A%2F%2Fapi.tiles.mapbox.com%5C%22%2Csource%3A%5C%22docs-examples%5C%22%7D)%3C%2Fscript%3E%20%3C%2Fhtml%3E%22%5D)

```suggestion
            const versionCSSRegex = /https:\/\/api\.mapbox\.com\/mapbox-gl-js\/v[0-9]+\.[0-9]+\.[0-9]+\/mapbox-gl\.css/g;
```